apps/site/assets/package.json

```diff
@@ -107,6 +113,7 @@
     "ts:build": "cd ts/ && tsc",
     "eslint": "cd js && npx eslint --ext .js .",
     "tslint": "cd ts && npx eslint --ext .ts,.tsx --max-warnings=0 .",
+    "stylelint": "npx stylelint css/**/*.scss --ignore-path .stylelintignore",
```

Comment:
`.stylelintignore` appears to be the default path for the ignore file; do we need to specify it here?

Reply:
It is the default, but for some reason it would never take, at least when I'd try to run the script. 😖

Reply:
Strange. I suppose it can't hurt to have it here.